packages/mgt-components/src/components/mgt-person/mgt-person.tests.ts

```diff
@@ -176,4 +191,97 @@ describe('mgt-person - tests', () => {
       })}' view="twoLines"></mgt-person>`);
     await expect(person.shadowRoot.querySelector('span.initials')).lightDom.to.equal('FV');
   });
+
+  it('should support a change in presence', async () => {
+    useConfig({
+      default: 'Available',
+      3: 'Busy',
+      4: 'Busy',
+      5: 'Busy'
+    });
+
+    PresenceService.config.initial = 1000;
+    PresenceService.config.refresh = 2000;
```

Comment:
perhaps we could use shorter timeouts here to reduce the test execution time?

Reply:
I ran a control set of 10 tests. I reduced to 100 and 200 ms instead and ran the test 10 times. This test passed all 20 times. I reduced to 10 and 20 ms and the test failed most of the time. I left it at 100 and 200 ms.

It is noteworthy that not all tests pass every time. Generally they do, but for example, "should render" failed on one of those 25 tests because it timed out. I just mention this to say, unit tests that depend on something happening in a period of time are not perfectly reliable.